CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v0.17.0
+
+This is mostly a bugfix release that fixes a couple of panics and supports additional docker image syntax.
+
+ - Avoid panicing on scratch images ([248](https://github.com/appc/docker2aci/pull/248)).
```

Comment:
`s/panicing/panicking/`

Reply:
It's this way in the PR but I can fix it. Thanks!